strings/names/first_female.txt

```diff
@@ -506,241 +825,2536 @@ Madilyn
+Oluchi
+Olympia
+Oppila
 Oralie
+Orbei
+Orenda
+Padmini
 Paige
 Paisley
+Pak
+Pala
+Pamuya
+Panagoita
+Pani
 Paola
+Pari
 Paris
 Patience
 Patricia
 Pauleen
 Payton
+Peace
+Pearl
+Pele
 Pene
 Penelope
 Peregrine
 Perla
+Petra
 Peyton
+Phajee
+Phan
+Pherenike
 Phoebe
 Phyliss
 Phyllida
 Phyllis
+Pilar
+Ping
 Piper
 Porsche
+Pouri
+Praitun
+Pratik
+Preap
 Presley
 Priscilla
+Priti
 Prosper
+Proxima
 Prue
+Purva
 Quanah
 Quiana
+Rabika
 Rachael
 Rachel
 Raegan
 Raelene
 Rain
+Rajani
+Rajanitha
 Randa
 Randal
+Randi
+Rani
+Rania
+Rashida
+Rashmi
+Ratu
+Rauna
 Reagan
 Rebecca
+Rebeka
 Rebekah
+Rebel
+Reem
 Reene
 Reese
+Reeta
 Renie
-Roxana
+Rewi
+Reyna
 Rhetta
+Rihaab
 Rihanna
 Riley
+Rima
+Rimas
+Rishmeka
+Ritsuko
+Rona
 Ronnette
+Roosa
 Rosa
 Rose
 Rosemary
+Rous
 Rowan
+Roxana
+Ru
 Ruby
+Ruhi
+Ruihi
 Ruth
+Ruza
+Ruzena
 Rylee
 Ryleigh
 Rylie
+Sa
+Saar
 Sabella
+Sabik
+Sabine
 Sabrina
 Sachie
 Sadie
+Safa
 Sage
+Sagira
+Saida
+Saimi
+Sajal
+Sakura
+Salani
+Salitia
 Sally
+Salma
+Salome
+Salwa
 Samantha
 Samara
 Sandra
+Sang
+Sanja
 Sara
 Sarah
 Sarai
+Saran
 Saranna
+Sarika
 Sasha
+Satna
+Satsobek
 Savanna
+Savannah
 Scarlett
 Selena
+Selima
+Sen
 Seneca
+Seo
 Serena
 Serenity
+Seti
+Setsu
+Sevil
+Sextia
+Shadia
+Shai
+Shaminder
+Shamsi
 Shana
-Shanika
 Shaneah
+Shanika
 Shannon
 Shantae
-Sherlyn
 Sharla
+Shauriana
 Shayla
+Shefali
 Shelby
 Sheri
 Sherie
 Sherill
+Sherlyn
 Sherri
+Shila
+Shir
+Shireen
+Shoken
+Shu
 Sienna
 Sierra
+Signy
+Sigrid
+Sihle
+Silfa
+Sima
+Siri
+Sisay
 Sissy
+Sitra
+Siv
+Skiftere
+Skuld
 Skylar
 Skyler
+Snaiborg
 Sofia
+Soheila
+Sok
+Solny
+Sona
+Sonia
+Sonja
 Sophia
+Soraya
+Sovbika
+Stanislava
 Star
+Stefanie
 Stella
 Steph
 Stephanie
 Stephany
+Sua
 Sue
 Sukie
+Sumana
 Summer
 Sunshine
+Suravi
 Susanna
 Susannah
 Suzan
 Suzy
+Sveta
+Svetlana
+Sybell
 Sydney
-Sydney
+Szarlota
+Taban
+Tablita
+Taka
+Tala
 Talia
+Tama
+Tamar
+Tamara
 Tamika
+Tane
 Tania
+Tanit
 Tansy
+Tapu
 Taryn
 Tatiana
+Tatjana
 Tatum
 Tatyana
+Tauret
 Taylor
 Teagan
+Telma
+Temulun
+Teodora
 Tessa
+Thaili
+Thalia
+Thando
+Thi
+Thorhalla
+Thu
+Thula
+Tien
 Tiffany
+Tilda
+Tina
+Tingting
+Tisa
 Tolly
+Ton
 Topaz
+Tordis
 Tori
+Touch
 Tracee
 Tracey
 Trinity
+True
+Tsomah
+Tsura
+Tumay
+Turid
+Tuwa
 Ulyssa
+Uraburu
+Uruewa
+Uta
+Vai
+Val
 Valentina
 Valeria
 Valerie
+Valeriya
+Vanda
 Vanessa
+Vani
+Vanita
+Vassia
+Vedi
+Verda
 Verna
 Veronica
+Vibia
 Victoria
+Vigdis
+Vika
+Vilma
+Vilna
 Vinnie
 Violet
+Violeta
+Viollca
+Viorica
 Vivian
 Viviana
 Vivyan
+Vixen
+Vritika
+Vyshu
+Wacu
+Wandia
+Wanika
+Washta
+Weeko
 Wendi
 Wendy
+Wi
 Willow
 Wisdom
+Wish
+Wren
+Wynfride
 Wynonna
+Xiaoyan
 Ximena
+Xinyi
+Xolani
+Ya
+Yael
+Yahita
+Yan
+Yanina
+Yasaman
 Yasmin
+Yasmine
+Yesim
+Yi
+Ying
+Yisa
+Ylenia
+Yobanna
+Yodman
 Yolanda
+Yoshe
 Ysabel
+Yuka
+Yumako
+Yutu
+Yvette
+Yvonne
+Zabel
+Zaheda
+Zamani
+Zaminy
+Zara
+Zawati
+Zee
+Zena
+Zero
+Zeynep
+Zheng
+Ziba
+Zimra
+Zineta
+Zinhle
 Zoe
 Zoey
-Zune
\ No newline at end of file
+Zofia
+Zubaira
+Zune
+Zwina
+Aada
```

Comment:
This part of the list is not sorted, but doing it manually is kind of too much of a chore.
It could be sorted with a little python script, for example.

Reply:
`sort first_female.txt -o first_female.txt` also works